validator/coordinator/src/main/java/tech/pegasys/teku/validator/coordinator/ValidatorApiHandler.java

```diff
@@ -188,6 +193,30 @@ public ValidatorApiHandler(
                     state -> getAttesterDutiesFromIndexesAndState(state, epoch, validatorIndexes)));
   }
 
+  @Override
+  public SafeFuture<Optional<SyncCommitteeDuties>> getSyncCommitteeDuties(
+      final UInt64 epoch, final List<Integer> validatorIndexes) {
+    if (isSyncActive()) {
+      return NodeSyncingException.failedFuture();
+    }
+    final SpecVersion specVersion = spec.atEpoch(epoch);
+    final UInt64 slot =
+        specVersion.getBeaconStateUtil().getEarliestQueryableSlotForTargetEpoch(epoch);
```

Comment:
This logic actually applies to attestations.  SyncCommittee can come from a much wider range of slots - most of the time we can just use the current head state which is always immediately available.  In some cases the head state may have rolled forward into the next sync committee period already and then we should just get the state from the first slot of the requested epoch - we know that's in the right sync committee period then.

The requirement is that the state we have is from the same or previous sync committee period (`SyncCommitteeUtil.computeSyncCommitteePeriod` converts epoch to sync committee period).